src/js/views/globals/modal/modal.hbs

```diff
@@ -11,7 +11,7 @@
 
 <div class="modal-footer" data-footer-region>
   <div class="u-float--right">
-    <a href="#" class="modal-cancel js-close">{{ cancelText }}</a>
-    <button type="submit" class="{{ buttonClass }} js-submit">{{ submitText }}</button>
+    <button class="modal-cancel button--text js-close">{{ cancelText }}</button>
```

Comment:
:+1:

Reply:
we can actually remove `.modal-cancel` here now.  I assume it's not used anywhere